### PR TITLE
Feature: Add CLI params for authentication

### PR DESCRIPTION
**Pull request by moorthi07**
Issue #7 dpd start with mongodb authentication fixed
Issue #8 version no. in package.json fixed.
Issue #4 Keygen works as expected.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "deployd-cli",
-  "version": "2.0.0",
+  "version": "0.0.2",
```
Comment (NicolasRitouet) [inline package.json]: Please, keep version 2.0.0

File: lib/cli/start.js [hunk 2]
```diff
@@ -62,45 +63,62 @@ var start = function(file) {
...
+
+
+    if (program.host) {
+      if (program.auth) {
+        var auth = program.auth.split(":");
+        username = auth[0];
+        password = auth[1];
+        setCredentials(username, password);
+      } else if (program.username || program.password) {
+        console.log(' program.password', program.password)
```
Comment (NicolasRitouet) [inline lib/cli/start.js]: do not output the password in the logs

File: lib/cli/start.js [hunk 3]
```diff
@@ -62,45 +63,62 @@ var start = function(file) {
...
-                            console.error('Password cannot be blank.');
-                            process.exit();
-                        }
-                    });
-                },
-                startup
-            );
+
+
+    // Added by PV
```
Comment (NicolasRitouet) [inline lib/cli/start.js]: what does that mean?

File: package.json (hunk 1 above)
Comment (moorthi07) [inline package.json]: ok got it

File: lib/cli/start.js (hunk 3 above)
Comment (moorthi07) [inline lib/cli/start.js]: just note that where I made the change

File: lib/cli/start.js (hunk 2 above)
Comment (moorthi07) [inline lib/cli/start.js]: great catch. I'll remove this line, was for test

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+language: node_js
```
Comment (NicolasRitouet) [inline .travis.yml]: can you use the same as `deployd`?
https://github.com/deployd/deployd/blob/master/.travis.yml

Comment (moorthi07) [inline .travis.yml]: Ok. Done.

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  directories:
+    - node_modules
+sudo: false
+services: mongodb
+node_js:
+  - "7"
+  - "6"
+  - "5"
+  - "4"
+- "0.12"
```
Comment (NicolasRitouet) [inline .travis.yml]: it needs a tab (you can validate your travis.yml here: http://lint.travis-ci.org/)

**Comment by NicolasRitouet** [comment]
@moorthi07 Thanks a lot for your contribution :tada:

**Comment by moorthi07** [comment]
Right back at you! thx.

**Pull request merged**